index.js

```diff
@@ -15,13 +15,30 @@ import RealmProvider from "providers/RealmProvider";
 import React from "react";
 import { AppRegistry } from "react-native";
 import Config from "react-native-config";
+import { setJSExceptionHandler, setNativeExceptionHandler } from "react-native-exception-handler";
 import { GestureHandlerRootView } from "react-native-gesture-handler";
 import { startNetworkLogging } from "react-native-network-logger";
 import { MD3LightTheme as DefaultTheme, Provider as PaperProvider } from "react-native-paper";
 import { SafeAreaProvider } from "react-native-safe-area-context";
 import colors from "styles/tailwindColors";
 
 import { name as appName } from "./app.json";
+import { log } from "./react-native-logs.config";
+
+const rootLog = log.extend( "root" );
```

Comment:
Like in the other PR, I'd do something like `const logger = log.extend( "index" );` or `const logger = log.extend( "index.js" );` so it's a little easier to find where these log statements are coming from.